### PR TITLE
[READY] Pride Month Holiday

### PR DESCRIPTION
**Pull request by jaw-sh**
Inspired by the /tg/station's Discord Icon
image

## About The Pull Request
Adds a Pride Month to the in-game holiday calendar.

## Why It's Good For The Game
Improves visibility of minority groups.

## Changelog
:cl:
add: Added a Pride Month event with flavor station names for June.
add: Added a Transgender Potion with custom assets all crew get during Pride Month.
/:cl:

image

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
I got a bad feeling about this one chief

File: code/modules/research/xenobiology/xenobiology.dm [hunk 1]
```diff
@@ -969,6 +969,38 @@
...
+	if(L.gender == FEMALE)
+		L.gender = MALE
+		L.visible_message("<span class='boldnotice'>[L] suddenly looks more masculine!</span>", "<span class='boldwarning'>You suddenly feel more masculine!</span>")
+	else
+		L.gender = FEMALE
+		L.visible_message("<span class='boldnotice'>[L] suddenly looks more feminine!</span>", "<span class='boldwarning'>You suddenly feel more feminine!</span>")
+
+	// spread awareness to a very serious issue
+	if(rand(0,100) <= 41)
+		addtimer(CALLBACK(L, /mob/living/carbon/proc/david_reimer, user), rand(60, 300))
```
Comment (LemonInTheDark) [inline code/modules/research/xenobiology/xenobiology.dm:994]: I got a very bad feeling about this one

**Comment by Doosteroni** [comment]
This great and all..... but the
Potion seems a bit......much

File: code/modules/research/xenobiology/xenobiology.dm [hunk 2]
```diff
@@ -969,6 +969,38 @@
...
+
+	L.regenerate_icons()
+	qdel(src)
+
+/mob/living/carbon/proc/david_reimer(mob/user)
+	user.visible_message("<span class='suicide'>[user] is suddenly coming to terms with a life full of regret! It looks like [user.p_theyre()] trying to commit suicide!</span>")
```
Comment (ArcaneMusic) [inline code/modules/research/xenobiology/xenobiology.dm:1000]: Are you TRYING to get the Repo killed again

File: code/modules/research/xenobiology/xenobiology.dm (hunk 1 above)
Comment (Qustinnus) [reply code/modules/research/xenobiology/xenobiology.dm:994]: uhoh thats gonna be a gitban

**Comment by nemvar** [comment]
Jannie moment

**Comment by optimumtact** [comment]
Jannie moment

**Comment by 81Denton** [comment]
Jannie moment

**Pull request closed without merging**